src/test/java/com/example/rets_api/integration/RetsControllerIntegrationTest.java

```diff
@@ -26,7 +30,7 @@ public class RetsControllerIntegrationTest {
     private TestRestTemplate restTemplate;
 
     @Test
-    public void creatingProperty() {
+    public void step10CreatingProperty() {
```

Comment:
why do you need to add order between the tests?

Reply:
because of the first test, we create a property and we tested it to have a id number '1'; we I tried to test everything together it tests delete test first and in delete test I create a property, so the id 1 is not anymore for the create test, and the test was failing

Reply:
You can just change the line 44 from RetsControllerIntegrationTest to assertNotNull(responseEntity.getBody()) so you don't need to verify the exactly value inside the body and don't need to add extra imports

Reply:
done :)